File: module_lib_player/src/main/java/org/live/player/littlevideo/MockHelper.java
package org.live.player.littlevideo;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by wl on 2018/12/5.
 */
public class MockHelper {

    public static List<String> getDatas() {
        List<String> datas = new ArrayList<>();
        for (int i = 0; i < 10; i++) {
            datas.add("menu: " + i);
        }
        return datas;
    }
}

File: module_test/src/main/java/org/live/test/test/MockHelper.kt
package org.live.test.test

import org.live.ui.MenuItem

/**
 * Created by wl on 2018/11/30.
 */
object MockHelper {

    @JvmStatic
    fun getMenuDatas(): MutableList<MenuItem> {
        val datas = mutableListOf<MenuItem>()
        for (i in 0..20) {
            val menuItem = MenuItem("menu: $i")
            datas.add(menuItem)
        }
        return datas
    }
}